File: tests/fixtures/py_named_imports/consumer.py
from mod import parse, helper as h


# use() appelle deux noms explicitement importes depuis mod :
#   - parse() -> mod.py::parse   (le nom importe contourne le filtre builtin)
#   - h()     -> mod.py::helper  (alias 'helper as h' ; PAS de fan-out vers other.py::helper)
def use():
    return parse() + h()

File: tests/fixtures/js_named_imports/consumer.ts
import { foo } from './mod';
import { parse } from './mod';
import { foo as bar } from './other';

// use() appelle trois noms importes :
//   - foo()   -> mod.ts::foo    (desambiguise : pas other.ts::foo)
//   - parse() -> mod.ts::parse  (le nom importe contourne le filtre builtin)
//   - bar()   -> other.ts::foo  (alias 'foo as bar' ; noeud = nom d'ORIGINE 'foo')
function use() {
  return foo() + parse() + bar();
}
